name_that_hash/prettifier.py

```diff
@@ -81,7 +81,7 @@ def pretty_print_one(self, objs: List, multi_print: bool):
             out += "\n[bold underline #5f5fff]Least Likely[/bold underline #5f5fff]\n"
 
             for i in rest:
-                out += self.turn_named_tuple_pretty_print(i) + " "
+                out += self.turn_named_tuple_pretty_print(i) + "\n"

```

Comment:
Here

Reply:
Fixed, replaced the newline with the space as per previous config.